<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiimoteSettingsPage</class>
 <widget class="QWidget" name="WiimoteSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wiimotedev</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/data.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="wiimotedev_enable">
     <property name="text">
      <string>Enable Wii Remote support</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>You can use your Wii Remote as a remote control for Clementine.  &lt;a href=&quot;http://www.clementine-player.org/wiimote&quot;&gt;See the page on the Clementine wiki&lt;/a&gt; for more information.
</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="wiimotedev_active">
        <property name="text">
         <string>Connect Wii Remotes using active/deactive action</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wiimotedev_focus">
        <property name="text">
         <string>Enable shortcuts only when Clementine is focused</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wiimotedev_notification">
        <property name="text">
         <string>Use notifications to report Wii Remote status</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Use Wii Remote</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wiimotedev_device">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Buttons</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Action</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="wiimotedev_add_action">
          <property name="text">
           <string>Add action</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="wiimotedev_delete_action">
          <property name="text">
           <string>Remove action</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="wiimotedev_reload">
          <property name="text">
           <string>Defaults</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>wiimotedev_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
